draft-ietf-tls-esni.md

```diff
@@ -665,46 +705,53 @@ the TLS connection to the backend server (if in Split Mode). In
 the latter case, it does not make any changes to the TLS
 messages, but just blindly forwards them.
 
-If the server sends a NewSessionTicket message, the corresponding ESNI PSK MUST
-be ignored by all other servers in the deployment when not negotiating ESNI,
+If the server sends a NewSessionTicket message, the corresponding ECHO PSK MUST
+be ignored by all other servers in the deployment when not negotiating ECHO,
 including servers which do not implement this specification (in Split mode,
-the server can detect this case by the presence of the "esni_info" extension).
+the server can detect this case by the presence of the "echo_info" extension).
```

Comment:
What's this extension? Doesn't seem to be defined anywhere.

Reply:
Good catch. I removed this parenthetical.